actionview/lib/action_view/flows.rb

```diff
@@ -15,7 +15,7 @@ def get(key)
 
     # Called by each renderer object to set the layout contents.
     def set(key, value)
-      @content[key] = value
+      @content[key] = ActiveSupport::SafeBuffer.new(value)
```

Comment:
I admit to not knowing much about this code, so forgive me if the following questions are missing something...

It seems from the comment associated with this method that it might impact a lot more than `content_for`. Would the following method, `append`, which says it's called by `content_for` be more appropriate?

Also, would this make all content provided to `content_for` html safe? How do we know this is safe?